pkg/cli/dump_test.go

```diff
@@ -490,3 +490,30 @@ ALTER TABLE orders VALIDATE CONSTRAINT fk_customer;
 		t.Fatalf("expected: %s\ngot: %s", want1, dump1)
 	}
 }
+
+func TestDumpingUUIDColumns(t *testing.T) {
+	// Testing use case described in #33664, that's it
+	// to create a table with UUID generated columns and
+	// then insert few rows. Dump table with the content
+	// and assert it can do it successfully.
+	defer leaktest.AfterTest(t)()
+
+	c := newCLITest(cliTestParams{t: t})
+	defer c.cleanup()
+
+	const create = `
+CREATE DATABASE foo;
+USE foo;
+CREATE TABLE bar(id UUID PRIMARY KEY DEFAULT gen_random_uuid(), name String);
+INSERT INTO bar (name) VALUES ('a'), VALUES ('b'), VALUES ('c');
```

Comment:
can you make this more rows:
1. turn `name` into an integer
2. `insert into bar(x) select generate_series(1,10000)`